network/netvisor/pn_ospf.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/python
+""" PN-CLI vrouter-ospf-add/remove """
+
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import shlex
+
+DOCUMENTATION = """
+---
+module: pn_ospf
+author: "Pluribus Networks (@amitsi)"
+version_added: "2.2"
+version: 1.0
+short_description: CLI command to add/remove ospf protocol to a vRouter.
+description:
+  - Execute vrouter-ospf-add, vrouter-ospf-remove command.
+  - This command adds/removes Open Shortest Path First(OSPF) routing
+    protocol to a virtual router(vRouter) service.
+options:
+  pn_cliusername:
+    description:
+      - Provide login username if user is not root.
+    required: False
+    type: str
```

Comment:
No need to document `type:` in the `options:` block.

`type:` *is* required in the `RETURN` section